aiounifi/interfaces/api_handlers.py

```diff
@@ -98,8 +98,15 @@ def __init__(self, controller: "Controller") -> None:
     @final
     async def update(self) -> None:
         """Refresh data."""
-        raw = await self.controller.request(self.api_request)
-        self.process_raw(raw.get("data", []))
+        data = await self.controller.request(self.api_request)
+        self.process_typed_api_response(data)
+
+    @final
+    def process_typed_api_response(self, response: TypedApiResponse) -> None:
+        """Process full typed API response"""
+        if "data" in response:
+            for response_item in response["data"]:
+                self.process_item(response_item)
```

Comment:
```suggestion
        for response_item in response.get("data", []):
            self.process_item(response_item)
```

Reply:
How could I miss that; that was so unpythonic of me :see_no_evil:

Reply:
I have been able to remove the 'process_typed_api_response', by calling process_raw with the correct argument.